### PR TITLE
[GEN-454] Gym/Daily analytics tab

### PR DESCRIPTION
**Pull request by redpepper-nag**
**Main changes**
- addition of a top navigator in AppNavigator that navigates between Daily summary and competition within Analytics
- renaming of tabs to be less confusing
- hardcoding of summary values to build UI

Screenshot_1702586632
Simulator Screenshot - iPhone SE (3rd generation) - 2023-12-14 at 15 43 47

### CODE REVIEW

File: trk/ios/Podfile.lock [hunk 1]
```diff
@@ -1608,9 +1615,9 @@ SPEC CHECKSUMS:
   React-RCTVibration: ea3a68a49873a54ced927c90923fc6932baf344a
   React-rncore: 9672a017af4a7da7495d911f0b690cbcae9dd18d
   React-runtimeexecutor: 369ae9bb3f83b65201c0c8f7d50b72280b5a1dbc
-  React-runtimescheduler: 2359c039f38bf2ea38ab22629d2f0bf53f05f628
-  React-utils: e9716fa24b691496e2a23650168a4840101ba895
-  ReactCommon: cd9b989bf8ac0219a081a1c915b8c38853a3dd37
+  React-runtimescheduler: 513d7b0c4292efc646fb66f414cbb52c665cf1b9
+  React-utils: 833a34b9c1cd1bec019361b704ac3128988c114e
+  ReactCommon: b75d2271cbb99f818df90d7a5cd830e0dc95c89a
   RecaptchaInterop: 7d1a4a01a6b2cb1610a47ef3f85f0c411434cb21
```
Comment (redpepper-nag) [inline trk/ios/Podfile.lock:1621]: this podfile is required because i imported a new library

File: trk/package.json [hunk 2]
```diff
@@ -35,10 +36,12 @@
...
     "react-native-nfc-manager": "^3.14.6",
+    "react-native-pager-view": "^6.2.3",
     "react-native-paper": "^5.11.1",
     "react-native-reanimated": "^3.4.2",
     "react-native-safe-area-context": "^4.7.1",
     "react-native-screens": "^3.24.0",
+    "react-native-tab-view": "^3.5.2",
     "react-native-vector-icons": "^10.0.0",
     "readable-stream": "^1.0.33",
     "stream-browserify": "^1.0.0",
```
Comment (redpepper-nag) [inline trk/package.json:47]: Same here, this is because of the new component import

File: trk/src/Navigation/AppNavigator.js [hunk 3]
```diff
@@ -14,10 +16,35 @@ import SetDetail from '../Screens/SetDetail';
...
+        options={{ title: 'Daily Summary' }}
+      />
+      <GymTopTab.Screen 
+        name="CompRanking"
+        component={CompRanking} 
+        options={{ title: 'Competition' }}
+      />
+    </GymTopTab.Navigator>
+  );
+}
```
Comment (redpepper-nag) [inline trk/src/Navigation/AppNavigator.js:48]: created a top tab navigator that swipes from left to right between the new screen 'Daily Summary' and the currently existing comp ranking

File: trk/src/Navigation/AppNavigator.js [hunk 4]
```diff
@@ -87,21 +114,22 @@ function ProfileStack() {
   );
 }
 
-function RankStack() {
-  console.log('[TEST] RankStack called');
+function AnalyticsStack() {
+  console.log('[TEST] AnalyticsStack called');
   return (
```
Comment (redpepper-nag) [inline trk/src/Navigation/AppNavigator.js:119]: renamed the entire stack to analyticsstack

File: trk/src/Navigation/AppNavigator.js [hunk 5]
```diff
@@ -87,21 +114,22 @@ function ProfileStack() {
...
     <Stack.Navigator>
       {/* Here, change the name of the screen to 'Competition_Ranking_screen' */}
       <Stack.Screen
-        name="Competition_Ranking_screen"
-        component={CompRanking}
-        options={{ title: 'Analytics', headerBackTitleVisible: false }}
+        name="MyGym"
+        component={GymTabs}
+        options={{ title: 'My Gym', headerBackTitleVisible: false }}
       />
```
Comment (redpepper-nag) [inline trk/src/Navigation/AppNavigator.js:126]: added the newly made top tab stack as a stack screen in the Analytics stack

File: trk/src/Navigation/AppNavigator.js [hunk 6]
```diff
@@ -130,9 +158,9 @@ function AppTabs() {
...
       {role === 'climber' ? null :
         <Tab.Screen
-          name="RankingTab"
-          component={RankStack}
-          options={{ title: 'Analytics', headerShown: false }}
+          name="AnalyticsTab"
+          component={AnalyticsStack}
+          options={{ title: 'My Gym', headerShown: false }}
         />}
```
Comment (redpepper-nag) [inline trk/src/Navigation/AppNavigator.js:164]: renamed the entire stack AnalyticsTab with a bottom tab name of 'My Gym'

File: trk/src/Screens/GymDaily/index.js [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+    
+  },
+  feedback: {
+    fontStyle: 'italic',
+    padding: 10,
+    color: 'black',
+  }
+});
+
+export default GymDaily;
```
Comment (redpepper-nag) [inline trk/src/Screens/GymDaily/index.js:146]: New screen that shows the summarised hardcoded data in box view

File: trk/yarn.lock [hunk 8]
```diff
@@ -6202,6 +6215,13 @@ react-native-screens@^3.24.0:
...
     warn-once "^0.1.0"
 
+react-native-tab-view@^3.5.2:
+  version "3.5.2"
+  resolved "https://registry.yarnpkg.com/react-native-tab-view/-/react-native-tab-view-3.5.2.tgz#2789b8af6148b16835869566bf13dc3b0e6c1b46"
+  integrity sha512-nE5WqjbeEPsWQx4mtz81QGVvgHRhujTNIIZiMCx3Bj6CBFDafbk7XZp9ocmtzXUQaZ4bhtVS43R4FIiR4LboJw==
+  dependencies:
+    use-latest-callback "^0.1.5"
+
 react-native-vector-icons@^10.0.0:
```
Comment (redpepper-nag) [inline trk/yarn.lock:6225]: required imports

**Review by VGmt — APPROVED** [review]

**Pull request merged**